hive-metastore/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -251,12 +251,12 @@ public void createNamespace(Namespace namespace, Map<String, String> meta) {
             namespace);
 
     } catch (TException e) {
-      throw new RuntimeException("Failed to create namespace " + namespace + " in Hive MataStore", e);
+      throw new RuntimeException("Failed to create namespace " + namespace + " in Hive MetaStore", e);
 
     } catch (InterruptedException e) {
       Thread.currentThread().interrupt();
       throw new RuntimeException(
-          "Interrupted in call to createDatabase(name) " + namespace + " in Hive MataStore", e);
+          "Interrupted in call to createDatabase(name) " + namespace + " in Hive MetaStore", e);
```

Comment:
If we go with `Metastore` and not `MetaStore` (which the former is correct according to a number of Hive and Cloudera docs), this also needs to be updated - plus several other locations :)